easybuild/easyconfigs/t/Tinker-HP/Tinker-HP-20241026-foss-2023a-plumed-colvars.eb

```diff
@@ -0,0 +1,60 @@
+easyblock = 'ConfigureMake'
+
+name = 'Tinker-HP'
+version = '20241026'
+versionsuffix = '-plumed-colvars'
+_commit = '5fa89f3'
+
+homepage = 'https://tinker-hp.org'
+description = """
+Tinker-HP: High-Performance Massively Parallel Evolution of Tinker on CPUs & GPUs"""
+
+toolchain = {'name': 'foss', 'version': '2023a'}
+toolchainopts = {'pic': True, 'usempi': True}
+
+github_account = 'TinkerTools'
+
+sources = [{
+    'source_urls': [GITHUB_SOURCE],
+    'download_filename': '%s.tar.gz' % _commit,
+    'filename': SOURCE_TAR_GZ,
+}]
+checksums = ['a92f8af90922a8980774c960c6846041ed25a719162d012f757308702de889ef']
+
+builddependencies = [
+    ('Autotools', '20220317'),
+    ('make', '4.4.1'),
+]
+dependencies = [
+    ('imkl', '2023.1.0'),
+    ('Tcl', '8.6.13'),
+]
+
+start_dir = 'v1.2'
+
+maxparallel = 1
+
+preconfigopts = "sed -i 's/-O3/-O3 -march=native/g' configure.ac && "
+preconfigopts += "sed -i 's/Wno-argument-mismatch/fallow-argument-mismatch/g' configure.ac && "
+preconfigopts += "autoreconf -fi && "
+
+configure_without_installdir = True
+
+configopts = '--with-blaslib=mkl --enable-fft-mkl --enable-plumed --enable-colvars'
+
+postinstallcmds = [
+    'cp -a bin params plumed/Gnu/* %(installdir)s',
+    r'echo -e "#!/usr/bin/env bash\n\$(basename \$0)_plumed_colvars_tcl.prod \$*" >%(installdir)s/bin/tinkerhp',
```

Comment:
this is weird, is making a wrapper script for `tinkerhp`? If that is the case, I prefer to use a plain patch file with the wrapper, like we do in VTK for `vtk-version.egg-info` or in Julia with `julia.wrapper`.